qulice-findbugs/src/main/java/com/qulice/findbugs/FindBugsValidator.java

```diff
@@ -176,14 +176,22 @@ private String excludes(final Environment env,
      * Creates xml with exclude patterns in findbugs native format.
      * @param excludes Exclude patterns
      * @return XML with findbugs excludes
+     * @todo #350 It's better to build XML using Xembly, but when it's
```

Comment:
@palyuga let's investigate... looks like some dependency is bringing a transitive one to the build. this "batik" is absolutely out of scope in qulice and can easily be removed (excluded)